db/db_impl/db_impl.cc

```diff
@@ -1948,12 +1955,6 @@ Status DBImpl::GetEntity(const ReadOptions& read_options,
         "Cannot call GetEntity without a PinnableWideColumns object");
   }
 
-  if (read_options.io_activity != Env::IOActivity::kUnknown) {
-    return Status::InvalidArgument(
-        "Cannot call GetEntity with `ReadOptions::io_activity` != "
-        "`Env::IOActivity::kUnknown`");
-  }
-
```

Comment:
[Note] Please ignore removal of check like this for GetEntity API or Multi/GetForUpdate in TXN related code API. I will either add back the check or include these API as enum in this PR.